### PR TITLE
[rails-6] fix rails-5 compatibility

### PR DESCRIPTION
**Pull request by wenn**

### CODE REVIEW

File: lib/rescue_unique_constraint.rb [hunk 1]
```diff
@@ -39,5 +43,12 @@ def rescue_unique_constraint(index:, field:)
       end
       index_rescue_handler.add_index(index, field)
     end
+
+    def rails_6?
+      byebug
```
Comment (ebenoist) [inline lib/rescue_unique_constraint.rb:48]: remove pls :)

**Review by ebenoist — APPROVED** [review]

File: lib/rescue_unique_constraint.rb (hunk 1 above)
Comment (wenn) [reply lib/rescue_unique_constraint.rb:48]: shooot

File: lib/rescue_unique_constraint.rb [hunk 2]
```diff
@@ -39,5 +43,11 @@ def rescue_unique_constraint(index:, field:)
       end
       index_rescue_handler.add_index(index, field)
     end
+
+    def rails_6?
+      Rails.gem_version.segments.first > RAILS_6_VERSION
```
Comment (mcostanza) [inline lib/rescue_unique_constraint.rb:48]: Should this be `>= RAILS_6_VERSION` ? `6 > 6` is going to return false?

Comment (ebenoist) [reply lib/rescue_unique_constraint.rb:48]: @mcostanza makes a very solid point

Comment (wenn) [reply lib/rescue_unique_constraint.rb:48]: yep.. test stubbing failed me. Thanks Mike

**Pull request merged**